02.recognize_digits/README.md

```diff
@@ -124,183 +136,179 @@ PaddlePaddle provides a Python module, `paddle.dataset.mnist`, which downloads a
 |t10k-labels-idx1-ubyte |  Evaluation labels | 10,000 |
 
 
+## Fluid API Overview
```

Comment:
This is good summarization. Should we include that in Chapter 1? @daming-lu

Reply:
Yes